src/operators.jl

```diff
@@ -708,6 +708,40 @@ egonet(g::AbstractGraph{T}, v::Integer, d::Integer, distmx::AbstractMatrix{U}=we
     g[neighborhood(g, v, d, distmx, dir=dir)]
 
 
+"""
+    induced_bipartite_subgraph(G,X,Y)
+
+Return the bipartite subgraph of `g` induced by the disjoint subsets of vertices X and Y.
+
+"""
+function induced_bipartite_subgraph(g::T,X::AbstractVector{U},Y::AbstractVector{U})  where T <: AbstractGraph where U <: Integer
```

Comment:
From convention it would be better to use `G` instead of `T` for the graph type. Furthermore, as far as I understand, this method would not work on any kind of `AbstractGraph`, only on `SimpleGraph` and `SimpleDiGraph`. So I would restrict it to these types of graphs.

Reply:
I copied the style of `induced_subgraph` that also uses `T` for the graph type. If this is more consistent with the convention, I can change it. Also, we could follow what's been done for `induced_subgraph` which also returns a mapping from the subgraph to the whole graph. This solution would let people use this function for any AbstractGraph (for example, using the mapping to retrieve information in a metagraph)

Reply:
I think following `induced_subgraph` is a good idea for the mapping, for the types I'd go with `G`